src/fides/api/service/connectors/saas_connector.py

```diff
@@ -476,7 +490,9 @@ def process_response_data(
                 postprocessor.strategy,  # type: ignore
             )
             try:
-                processed_data = strategy.process(processed_data, identity_data)
+                processed_data = strategy.process(
+                    processed_data, identity_data, param_values
+                )
```

Comment:
Let's pass in the `privacy_request` in here instead. Privacy request has a `get_raw_access_results()` function that we can use to get a map of all the results we have up until now.

```suggestion
                processed_data = strategy.process(
                    processed_data, identity_data, privacy_request
                )
```